build: Bump containerd to v1.6.4 and pause to 3.7

gdemonet opened the pull request:
See https://github.com/containerd/containerd/releases/tag/v1.6.4
and https://github.com/kubernetes/kubernetes/blob/master/build/pause/CHANGELOG.md

gdemonet committed:
build: Bump containerd to v1.6.4

See https://github.com/containerd/containerd/releases/tag/v1.6.4

gdemonet committed:
build: Bump pause image to 3.7

See https://github.com/kubernetes/kubernetes/blob/master/build/pause/CHANGELOG.md

bert-e commented:
# Hello gdemonet,



My role is to assist you with the merge of this
pull request. Please type `@bert-e help` to get information
on this process, or consult the [user documentation](
https://eve.devsca.com/github/scality/metalk8s/bert-e/doc/user).





*Status report is not available.*

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* one peer





Peer approvals *must* include at least 1 approval from the following list:

* @alexandre-allard-scality

* @jbwatenbergscality

* @nicolast

* @gdemonet

* @teddyandrieux

* @thomasdanan

* @lucieleonard

* @chengyanjin

TeddyAndrieux commented on packages/redhat/common/containerd.spec:
```diff
@@ -58,7 +58,7 @@ Requires:       runc
 # vendored libraries
 # From github.com/containerd/containerd repository, checkout the corresponding version and run:
-# go list -m -f '{{ if not .Main }}{{.Path }} {{ .Version }}{{ end }}' -u all 2> /dev/null \
+# go list -m -mod=mod -f '{{ if not .Main }}{{.Path }} {{ .Version }}{{ end }}' -u all 2> /dev/null \
```
:+1:

gdemonet committed:
ci: Increase timeout to 40mn for a doit build

It happens sometimes that for some unknown reason, downloading very
large files (such as Calico RPM sources) will hang for a long time, and
Eve kills the build after 20mn without output. We raise this timeout to
40mn, to see if that helps.

gdemonet commented:
/approve

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* one peer





Peer approvals *must* include at least 1 approval from the following list:

* @alexandre-allard-scality

* @jbwatenbergscality

* @nicolast

* @gdemonet

* @teddyandrieux

* @thomasdanan

* @lucieleonard

* @chengyanjin










*The following options are set:* **approve**

bert-e commented:
# In the queue



The changeset has received all authorizations and has been added to the
relevant queue(s). The queue(s) will be merged in the target development
branch(es) as soon as builds have passed.

The changeset will be merged in:


* :heavy_check_mark: `development/123.0`



The following branches will **NOT** be impacted:

* `development/2.0`
* `development/2.1`
* `development/2.10`
* `development/2.11`
* `development/2.2`
* `development/2.3`
* `development/2.4`
* `development/2.5`
* `development/2.6`
* `development/2.7`
* `development/2.8`
* `development/2.9`



There is no action required on your side. You will be notified here once
the changeset has been merged. In the unlikely event that the changeset
fails permanently on the queue, a member of the admin team will
contact you to help resolve the matter.

*IMPORTANT*

Please do not attempt to modify this pull request.

* Any commit you add on the source branch will trigger a new cycle after the
  current queue is merged.
* Any commit you add on one of the integration branches will be **lost**.

If you need this pull request to be removed from the queue, please contact a
member of the admin team now.




*The following options are set:* **approve**

bert-e commented:
# 


I have successfully merged the changeset of this pull request
into targetted development branches:


* :heavy_check_mark: `development/123.0`



The following branches have **NOT** changed:

* `development/2.0`
* `development/2.1`
* `development/2.10`
* `development/2.11`
* `development/2.2`
* `development/2.3`
* `development/2.4`
* `development/2.5`
* `development/2.6`
* `development/2.7`
* `development/2.8`
* `development/2.9`



Please check the status of the associated issue None.

Goodbye gdemonet.
